Source/WebKit/WPEPlatform/wpe/WPESettings.h

```diff
@@ -162,7 +162,48 @@ typedef enum {
  * Default: 1200
  */
 #define WPE_SETTING_CURSOR_BLINK_TIME "/wpe-platform/cursor-blink-time"
-
+/**
+ * WPE_SETTING_TOPLEVEL_WIDTH:
+ *
+ * The width of the toplevel size.
```

Comment:
And we need to document that this is the default width of the toplevel used at construction.